GravatarApp/Share/ShareFields/ShareFieldsSelectionStore.swift

```diff
@@ -49,7 +52,18 @@ class ShareFieldsSelectionStore: ObservableObject {
         didSet { profileURLStore = profileURL }
     }
 
-    init() {
+    func account(_ verifiedAcctoun: VerifiedAccount) -> Bool {
+        UserDefaults.standard.bool(forKey: verifiedAcctoun.url)
+    }
+
+    func set(_ verifiedAcctoun: VerifiedAccount, to value: Bool) {
+        UserDefaults.standard.set(value, forKey: verifiedAcctoun.url)
+        objectWillChange.send()
+    }
```

Comment:
The verified accounts have a different initial value(false) than others(true).

Also, the userDefault should better be injectable for unit testing. And using `@AppStorage` isn't helping us about it. This is not a View so it's not providing us auto-ui updates either. No need to use `@AppStorage` here.

I think we should just use the userDefaults directly here, solves all these things. I. made a PR for it:

https://github.com/Automattic/Gravatar-iOS/pull/64

Reply:
Yes I had this same feeling when I added user defaults for the verified accounts. I didn't put the time on it to think about how to implement the change.

Using user defaults directly totally looks like a better option. Thank you for working it out 🙏